modules/front-end/src/app/core/components/access-token-drawer/access-token-drawer.component.ts

```diff
@@ -94,6 +94,10 @@ export class AccessTokenDrawerComponent {
       name: [name, [Validators.required], [this.nameAsyncValidator], 'change'],
       type: [type, [Validators.required]]
     });
+
+    this.form.get('type').valueChanges.subscribe(()=>{
+      this.onTypeChange();
+    })
   }
```

Comment:
There is a bug here but not your problem, we need to change to this

```typescript
this.form.get('type').valueChanges.subscribe(newType => {
  this.isServiceAccessToken = newType === AccessTokenTypeEnum.Service;
});
```

And then delete the `onTypeChange` method

Reply:
Understood and made the changes.